LocalAtBrown/article-recommendations
# fix(widget): refactor plugin for canary deploy

### All Submissions:






### Changes proposed in this Pull Request:



Fixes the constructed data attributes in the recent posts function and recommendation function.

Also fixes the nesting of the recommendations and recent posts unordered lists so that the AMP experiments can select them correctly.

### How to test the changes in this Pull Request:

1. Build the plugin and upload to WP
2. Set the widget in sidebar from widget settings
3. Open a post
4. You should see article recommendations and recent posts both displayed.

### Other information:
